docs/run-nym-nodes/nodes/mixnodes.md

```diff
@@ -142,14 +163,16 @@ Initalise your mixnode with the following command, replacing the value of `--id`
 ./nym-mixnode init --id winston-smithnode --host $(curl ifconfig.me) --wallet-address <wallet-address>
 ```
 
+Your `--host` must be publicly routable on the internet in order to mix packets, and can be either an Ipv4 or IPv6 address. 
+
 :::caution
-Please make sure you have access to the `--wallet-address` account, and download the Desktop Wallet [here](https://nymtech.net/get-involved) if you have not already done so, in order to be able to interact with your node!
+Your node **must** be able to send TCP data using **both** IPv4 and IPv6 (as other nodes you talk to may use either protocol).
 :::
 
-Your `--host` must be publicly routable on the internet in order to mix packets, and can be either an Ipv4 or IPv6 address. Your node _must_ be able to send TCP data using _both_ IPv4 and IPv6 (as other nodes you talk to may use either protocol). The `$(curl ifconfig.me)` command above returns your IP automatically using an external service. Alternatively, you can enter your IP manually wish. If you do this, remember to enter your IP **without** any port information.
+The `$(curl ifconfig.me)` command above returns your IP automatically using an external service. Alternatively, you can enter your IP manually wish. If you do this, remember to enter your IP **without** any port information.
 
-:::caution
-Please note that the `init` command will refuse to destroy existing mixnode keys.
+:::note
+The `init` command will refuse to destroy existing mix node keys.
```

Comment:
I noticed you changed all mixnode to mix node

Reply:
yeah just the ones not in code blocks, to get docs in line with the blogposts